app/app.go

```diff
@@ -408,10 +408,10 @@ func New(
 	// Create Transfer Keepers
 	app.TransferKeeper = ibctransferkeeper.NewKeeper(
 		appCodec, keys[ibctransfertypes.StoreKey], app.GetSubspace(ibctransfertypes.ModuleName),
+		app.IBCKeeper.ChannelKeeper, // may be replaced with middleware such as ics29 fee
 		app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
 		app.AccountKeeper, app.BankKeeper, scopedTransferKeeper,
 	)
```

Comment:
From IBC v3, [IBC middleware](https://ibc.cosmos.network/main/ibc/middleware/develop.html) has been introduced.

As above, if we don't want to use any middleware for now, we can just pass the `app.IBCKeeper.ChannelKeeer` as a `ICS4Wrapper`. It's explained in the [IBC v2->v3 migration guide](https://ibc.cosmos.network/main/migrations/v2-to-v3.html#ics20). Also, I referenced the [ibc-go simapp](https://github.com/cosmos/ibc-go/commit/e40cdec6a#diff-d18972debee5e64f16e40807b2ae112ddbe609504a93ea5e1c80a5d489c3a08aR344).

Later, when we want to use any IBC middleware such as [IBC Fee middleware](https://ibc.cosmos.network/main/migrations/v3-to-v4.html#ics29-fee-middleware) like [this simapp](https://github.com/cosmos/ibc-go/pull/276/files#diff-d18972debee5e64f16e40807b2ae112ddbe609504a93ea5e1c80a5d489c3a08aR364).